src/slider/stateful-slider-container.js

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import React from 'react';
+import {STATE_CHANGE_TYPE} from './constants';
+import type {
+  StatefulContainerPropsT,
+  StateReducerT,
+  StateT,
+  ParamsT,
+  ChangeActionT,
+} from './types';
+
+const defaultStateReducer: StateReducerT = (type, nextState) => nextState;
+
+class StatefulSliderContainer extends React.Component<
+  StatefulContainerPropsT,
+  StateT,
+> {
+  static defaultProps = {
+    initialState: {},
+    stateReducer: defaultStateReducer,
+    onChange: () => {},
+  };
+
+  state = {...this.props.initialState};
```

Comment:
Should there be any initial value state?

Reply:
yes. It has to be `value` here. fixed it

Reply:
The fix seems to be lost as the line remained unchanged, so we still don't set the initial value here.